server/balancer.go

```diff
@@ -370,3 +372,262 @@ func (r *replicaChecker) checkBestReplacement(region *RegionInfo) Operator {
 	}
 	return newTransferPeer(region, oldPeer, newPeer)
 }
+
+// RegionStat records each hot region's statistics
+type RegionStat struct {
+	RegionID       uint64    `json:"region_id"`
+	WrittenBytes   uint64    `json:"written_bytes"`
+	HotDegree      int       `json:"hot_degree"`
+	LastUpdateTime time.Time `json:"last_update_time"`
+	StoreID        uint64    `json:"-"`
+	antiCount      int
+	version        uint64
+}
+
+// ListRegionsStat is a list of a group region state type
+type ListRegionsStat []RegionStat
+
+func (m ListRegionsStat) Len() int           { return len(m) }
+func (m ListRegionsStat) Swap(i, j int)      { m[i], m[j] = m[j], m[i] }
+func (m ListRegionsStat) Less(i, j int) bool { return m[i].WrittenBytes < m[j].WrittenBytes }
+
+// StoreHotRegions records all hot regions in one store with sequence
+type StoreHotRegions struct {
+	TotalWrittenBytes uint64          `json:"total_written"`
+	RegionCount       int             `json:"region_count"`
```

Comment:
we can see count directly in api more easy to know how many hot regions in that store.